pkg/backup/backup.go

```diff
@@ -427,8 +428,15 @@ func (kb *kubernetesBackupper) BackupWithResolvers(log logrus.FieldLogger,
 	updated.Status.Progress.TotalItems = len(backupRequest.BackedUpItems)
 	updated.Status.Progress.ItemsBackedUp = len(backupRequest.BackedUpItems)
 
+	// update the hooks execution status
+	if updated.Status.HookStatus == nil {
+		updated.Status.HookStatus = &velerov1api.HookStatus{}
+	}
+	updated.Status.HookStatus.HooksAttempted, updated.Status.HookStatus.HooksFailed = itemBackupper.hookTracker.Stat()
+	log.Infof("hook Tracker: %+v", itemBackupper.hookTracker.GetTracker())
```

Comment:
This should be fine for now, but my slight worry is this log will come very frequently, every time status is updated.
perhaps make it such that this log is skipped if no hooks are executed. The struct is empty.
To reduce the log frequency.